### PR TITLE
Speed up. Added beter error printing 

### PR DESCRIPTION
**Pull request by KyrylR**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -14,6 +14,8 @@ const storageSave: ActionType<CompareArgs> = async (taskArgs, env) => {
   mergeCompareArgs(env, taskArgs);
 
   // Make sure that contract artifacts are up-to-date.
+  await env.run(TASK_CLEAN);
```
Comment (Arvolear) [inline src/index.ts]: This is a rather unobvious thing. Many plugins override this task and seeing something disappear would be quite intriguing...

**Comment by Arvolear** [comment]
Versions?

File: src/storage/StorageLayout.ts [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+      return ParseBuildInfo(contract);
+    });
+
+    return {
+      buildInfos: artifacts,
+      inheritanceImpact: inheritanceParser.result,
+    };
+  }
+
+  private resolvePathToFile(path_: string, file_: string = ""): string {
```
Comment (Hrom131) [inline src/storage/StorageLayout.ts:104]: Will this solution work on Windows? It also uses other separators in the paths

File: src/storage/parsers/parsers.ts [hunk 3]
```diff
@@ -1,43 +1,30 @@
...
-      (a.source + ":" + a.name).localeCompare(b.source + ":" + b.name)
-    ),
+    contracts: contractStorageLayout,
     format: contract._format,
     solcLongVersion: contract.solcLongVersion,
     solcVersion: contract.solcVersion,
   };
 }
 
 function parseContracts(contracts: { [p: string]: { [p: string]: CompilerOutputContract } }): ContractStorageLayout[] {
```
Comment (Hrom131) [inline src/storage/parsers/parsers.ts]: Let's declare types for the parameters of these functions?

File: src/storage/StorageLayout.ts (hunk 2 above)
Comment (KyrylR) [reply src/storage/StorageLayout.ts:104]: Yep, it works

**Review by Hrom131 — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -1,13 +1,14 @@
 {
   "name": "@dlsl/hardhat-smart-compare",
-  "version": "1.0.0-alpha.1",
-  "description": "This package facilitates contract upgradeability and provides various comparison tools.",
+  "version": "1.0.0",
+  "description": "This package facilitates contract upgradability and provides various comparison tools.",
```
Comment (Arvolear) [inline package.json]:
```suggestion
  "description": "Compare contracts between upgrades to ensure compatibility",
```

**Pull request merged**